recipes/pycalphad/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set version = "0.4.1" %}
+{% set sha256 = "2603069927f4e6f69038f90a993f810c63585b6bf428473bcd5254d940a449f" %}
```

Comment:
Alternatively, I inspected stdout from the CI systems - it should be `d2603069927f4e6f69038f90a993f810c63585b6bf428473bcd5254d940a449f`.